### DISCUSSION TITLE
Synology Startup Help

### DISCUSSION THREAD

**Discussion by bzbybt** [AdrienPoupa/docker-compose-nas#56] [Q&A]
Hi Adrien,

Thank you for creating such a wonderful guide that is easy to follow. I have a similar setup with Docker containers on my DSM 7.2 device (most *arrs, PIA wireguard VPN, Homepage, etc). However, I am lost about how to get started to update my setup. Some background: I use docker compose for my containers but not familiar with using an environment file. Also my Docker config files for each container are on separate NVME SSDs (volume7) for speed and the data/download directory is on SATA HDDs (volume1) for storage. Here are my questions:

1. Quick Start: where do I save your files in my DSM? Does ./update-config.sh create issues with my existing setup and how do I edit them? I like to get app based URLs but don't want to mess up existing connections. Plus, I already have API keys and don't want to regenerate them if possible.
2. .env.example: what should I focus on here? Can I ignore dockers I don't use such as Jellyfin, Jellyseer, and Lidarr? 
3. docker-compose.yml: I am unfamiliar with Traefik but would like to replace the native synology Reverse Proxy server with this. Are there any things I should watch out for?

Thank you in advance!

**Comment by AdrienPoupa** [comment]
Hello,

You're welcome, I'm glad to hear that!

1. In my Synology setup, I've put the `docker-compose-nas` folder in a volume, it does not really matter where it is stored. To make the migration as clean as possible, I would recommend starting over. You can backup Sonarr, Radarr, Prowlarr from the UI, install Docker Compose NAS and restore your backups to the new installation. This means API keys will be regenerated, but is that an issue?
2. Yes, feel free to ignore anything that is not used.
3. Synology has some quirks, for example it takes over the ports 80 and 443 that are necessary for Traefik. To fix this and a few other issues, there's a whole section of the readme to follow: https://github.com/AdrienPoupa/docker-compose-nas?tab=readme-ov-file#synology-quirks

Also I see that you want to separate your downloads and your long-term storage, so you may want to follow this: https://github.com/AdrienPoupa/docker-compose-nas?tab=readme-ov-file#use-separate-paths-for-torrents-and-storage

**Reply by bzbybt** [reply]
Thanks for the feedback. 1 and 2 makes sense. I also have my downloads and long-term storage separated but in the same file structure like your example. For 3, I tried using the boot-up script. It would run but then Synology would reset it's web server and reserved ports 80 and 443 again. I'll try another solution.
